src/bin/pg_autoctl/cli_common.c

```diff
@@ -217,6 +219,39 @@ cli_create_node_getopts(int argc, char **argv,
 				break;
 			}
 
+			case 'V':
```

Comment:
commented list of options in the function header should be updated accordingly for this function and the other modified functions.

Reply:
Good point, done now!